.github/workflows/config.yml

```diff
@@ -1,513 +1,552 @@
-name: test-and-deploy
+name: refine.bio CI/CD
+
+env:
```

Comment:
This is minor but I think the order of `env` and `on` should be switched for clarity.

Reply:
Done.